xml/System.Net.Mail/SmtpClient.xml

```diff
@@ -53,7 +53,7 @@
   
  To include an attachment with an email message, first create the attachment by using the <xref:System.Net.Mail.Attachment> class, and then add it to the message by using the <xref:System.Net.Mail.MailMessage.Attachments%2A?displayProperty=nameWithType> property. Depending on the email reader used by the recipients and the file type of the attachment, some recipients might not be able to read the attachment. For clients that cannot display the attachment in its original form, you can specify alternate views by using the <xref:System.Net.Mail.MailMessage.AlternateViews%2A?displayProperty=nameWithType> property.  
   
- You can use the application or machine configuration files to specify default host, port, and credentials values for all <xref:System.Net.Mail.SmtpClient> objects. For more information, see [&lt;mailSettings&gt; Element (Network Settings)](~/docs/framework/configure-apps/file-schema/network/mailsettings-element-network-settings.md).  
+ In .NET Framework you can use the application or machine configuration files to specify default host, port, and credentials values for all <xref:System.Net.Mail.SmtpClient> objects. For more information, see [&lt;mailSettings&gt; Element (Network Settings)](~/docs/framework/configure-apps/file-schema/network/mailsettings-element-network-settings.md). But in .NET Core we don't support setting defaults. As a workaround you need to set the relevant properties on SmtpClient directly.  
```

Comment:
- comma after .NET Framework
- The .NET Core sentence should be rephrased to remove "But" and "we": ".NET Core does not support setting defaults. As a workaround, you must set the relevant properties on SmtpClient directly.  
- fence SmtpClient (\`SmtpClient\`)

Reply:
thanks @rpetrusha . How do I fence SmtpClient?
this way: `SmtpClient`
or 
this way: <xref:System.Net.Mail.SmtpClient>